lib/geordi/commands/dump.rb

```diff
@@ -36,10 +42,12 @@ def dump(target = nil, *_args)
 
       loader = DumpLoader.new(options.load)
 
-      Interaction.announce "Sourcing dump into the #{loader.config['database']} db"
+      database = options[:database] ? " #{options[:database]}" : ''
+
+      Interaction.announce "Sourcing dump into the #{loader.config['database']}#{database} db"
       loader.load
 
-      Interaction.success "Your #{loader.config['database']} database has now the data of #{options.load}."
+      Interaction.success "Your #{loader.config['database']}#{database} database has now the data of #{options.load}."
```

Comment:
These changes do not seem correct to me. They claim a dump was loaded to eg. "development primary", but that is not true. Further, we're in the "no target" (= development) logic branch, which you did not even change.

You should instead change L59, remote.rb:50 and L68 as required by the updated tests.

Reply:
I have to admit I don't fully understand what you meant in your first review with 
> Please add support for making a dump with multiple local databases by enhancing dump.rb:46
This was my confused interpretation of it.

Reply:
The dump command boils down to:
```
if target.nil? # No Capistrano stage given, so this run is about development
  if options.load # *Load* dump mode
     # load a dump file into the development database (1)
  else # Dump mode
    # make development dump (2)
else # Capistrano stage given (usually staging or production)
  # dump the given stage and download the dump (3)
  if options.load # loading the dump required
    # load the downloaded dump into the local database (3b)
  end
end
```

I was talking about the "make development dump" (2) part. When supporting to dump one of multiple remote databases (3), Geordi should equally be able to dump one of multiple local databases.

Reply:
> You should instead change L59, remote.rb:50 and L68 as required by the updated tests.

By this I meant that the output of step (3) and (3b) should be updated instead.